packages/venia-ui/lib/targets/makeRoutesTarget.js

```diff
@@ -11,11 +11,17 @@ function makeRoutesTarget(venia) {
 function addRoutes(routeList, routes) {
     for (const route of routes) {
         const AddedRoute = routeList.addReactLazyImport(route.path, route.name);
+        const exact = route.exact ? 'exact ' : '';
+        const authed = route.authed ? 'authed ' : '';
+        const path = JSON.stringify(route.pattern);
+        const redirectTo = route.redirectTo
+            ? JSON.stringify(route.redirectTo)
+            : null;
+        const redirectToProp = redirectTo ? `redirectTo={${redirectTo}} ` : '';
+
         routeList.prependJSX(
             'Switch',
-            `<Route ${route.exact ? 'exact ' : ''}path={${JSON.stringify(
-                route.pattern
-            )}}><${AddedRoute}/></Route>`
+            `<AuthRoute ${exact}${authed}${redirectToProp}path={${path}}><${AddedRoute}/></AuthRoute>`
```

Comment:
In here I would expect to see the logic to either render `<Route>` or `<AuthRoute>` depending on the `authed` prop.